### PR TITLE
doc: add Upload logs to archive server section...

### PR DESCRIPTION
**Pull request by smithfarm**
...to Testing in the cloud testing chapter

Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: doc/dev/index.rst [hunk 1]
```diff
@@ -1274,6 +1274,23 @@ there::
...
+will cause logs from all failed jobs to be uploaded to the log archive server
+maintained by the Ceph project. The logs will appear at the URL::
+
+    http://qa-proxy.ceph.com/teuthology/$RUN
+
+where ``$RUN`` is the name of the run - which will be something like
+"ubuntu-2016-07-23_16:08:12-rados-hammer-backports---basic-openstack"
+
+If you don't providing the ``--upload`` option, all the logs can still be found
+on the teuthology VM in the directory ... (?)
```
Comment (smithfarm) [inline doc/dev/index.rst]: @dachary Do you know where the daemon (OSD, MON, MDS) logs are stored if --upload is not given?

Comment (dachary) [inline doc/dev/index.rst]: On the teuthology machine

Comment (smithfarm) [inline doc/dev/index.rst]: haha - it tells me:

```
# logs in /usr/share/nginx/html
```

But the disk space on the teuthology machine is limited, eh?

**Comment by smithfarm** [comment]
@dachary I gave the `--upload` option with a run called

```
ubuntu-2016-07-23_21:53:47-rados:singleton-hammer-backports---basic-openstack
```

and now I'm trying to find it. I've tried qa-proxy.ceph.com and teuthology-logs.public.ceph.com but no luck so far.

**Comment by smithfarm** [comment]
@dachary Could you take a look at http://tracker.ceph.com/issues/16816 and clarify if necessary?

**Comment by smithfarm** [comment]
Since the public log server might be down for whatever reason, it would be useful to have a section explaining how to set up one's own log archive server.

**Comment by smithfarm** [comment]
@dachary Can you take another look?

**Pull request merged**